### DISCUSSION TITLE
General Support needed

### DISCUSSION THREAD

**Discussion by info-sic** [pandora-analysis/pandora#432] [Q&A] [answered]
Hi all,
first and foremost: thank you for this project. I'm setting up an instance for my workplace and do have some questions. Alas, I'm not a webdeveloper and was unable to figure out the answers by looking at the code. So hopefully this is the place to ask? Otherwise please lead me to the right place. And yes, I rtfm, everything I could find on the website and in GitHub.
1. Where is generic.conf to add admin users?
2. what are the default file permissions in the /pandora/* subtree? I needed to change them to 755 and forgot to backup them
3. How can I make sure, that analyzed files are securely deleted after analysis? Are there caches and logs to clear also? I want as few traces of files as possible.

TIA
Manu

**Comment by Rafiot** [comment, marked as answer]
Hi,

Yep, it's the right place.

1. The admin account(s) can be setup in the key `users` (format: `"admin": "password"`) of `config/generic.json` (see: https://github.com/pandora-analysis/pandora/blob/main/config/generic.json.sample#L17)
2. It doesn't really matters, as long as all the files are readable (and directories can be listed) by the user you use to run the `start` script.
3. There is no easy way to do that. The project was created in a way that an admin can access historical files. 
* The files themselves are stored in `tasks/`
* The logfiles are in `logs/*` and `website/logs/*`, but I don't think there is anything super relevant there
* The analysis results that contain what you see on the web interface are in a kvrocks DB in `storage/`, and it will contain text dumps from the uploaded files, hashes, things like that.

For the files themselves, I need to add a cleanup script (https://github.com/pandora-analysis/pandora/issues/431) so that could be used if you run it every 10 min or so to delete all the files. Regarding the analysis results stored in kvrocks, I didn't plan to get rid of them, but it could be a feature to the cleanup script (remove files only / remove files and tasks results).
Keep in mind that if you remove the results, everything will be gone, there will be nothing left and you cannot see the result either anymore.

Can you expand  bit on your needs there? What do you mean by `as few traces of files as possible`, and what do you still want to do with an analysis after cleanup (if anything).

**Comment by info-sic** [comment]
Hi Rafiot,
wow, that's what I call "quick support". Thanks for the help. Maybe update /doc/Administration Guide.md because the nonexistent "generic.conf" is mentioned there.
CU
Manu

**Reply by Rafiot** [reply]
Woops, good catch, fixed it!
